index.html

```diff
@@ -428,8 +426,10 @@
     <h2>Processing</h2>
     <section data-link-for="PerformanceObserver">
     <h2>Queue a <code>PerformanceEntry</code></h2>
-    <p>To <dfn>queue a PerformanceEntry</dfn> (<i>new entry</i>), run these
```

Comment:
We can't remove this, as this formally defines the step that we refer to from other specs.

To <dfn>queue a PerformanceEntry</dfn> (<var>new entry</var>) with an optional <var>add to performance entry buffer</var> flag, which is set to false(?) by default, ...

(let's use more descriptive names to make algorithm easier to read)